4.LastFM/js/lastfmLoader.js

```diff
@@ -0,0 +1,32 @@
+function LastfmLoader(param) {
+  this.params = param || {};
+  this.params.key = this.params.key || 'ea9abab35aef9b1a492f071c30b761b2';
+  this.params.url = this.params.url || 'http://ws.audioscrobbler.com/2.0/?';
+  this.params.format = this.params.format || 'json';
+  this.params.method = this.params.method || 'chart.getTopArtists'; //random stuff
+  this.getQuery = function(){};
```

Comment:
also please read about https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
it can be here